Nhaimerl voting tutorial (#2758)
dfinity/portal: docs/developer-docs/defi/rosetta/icp_rosetta/construction_api/voting/index.mdx

---
keywords: [intermediate, rosetta, tutorial]
---

import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";

> # NNS Voting

Reviewer: ```suggestion
# NNS voting
```

> This section will give you an overview of how how voting on the Neurons using Rosetta works. 

Reviewer: ```suggestion
This section will give you an overview of how to vote on NNS proposals using Rosetta. 
```

> This guide assumes you are familiar with the [flow of operations](/docs/developer-docs/defi/rosetta/icp_rosetta/construction_api/operations_flow/index.mdx). 
> Futhermore, it assumes that you have a Rosetta instance running at: `http://127.0.0.1:8081`.
> This tutorial also assumes that you are familiar with the concept of staking and voting on the NNS in general. To read up on how these concepts work on the Internet Computer Protocol
> you can look at the [documentation of the NNS](/docs/current/developer-docs/daos/nns/overview). 

Reviewer: ```suggestion
This tutorial assumes the following:

- You are familiar with the [flow of operations](/docs/developer-docs/defi/rosetta/icp_rosetta/construction_api/operations_flow/index.mdx). 

- You have a Rosetta instance running at: `http://127.0.0.1:8081`.

- You are familiar with the concept of staking and voting on the NNS in general. To read up on how these concepts work,
you can look at the [NNS documentation](/docs/current/developer-docs/daos/nns/overview). 
```
